SUMMARY.md

```diff
@@ -2,81 +2,107 @@
+  * [Nunjucks Helper Functions](tutorials/core-concepts/09-working-with-templates/nunjucks-helper-functions.md) 
+  * [Accessing images inside related pages](tutorials/core-concepts/09-working-with-templates/thumbnails-and-areas-of-child-pages.md)
+  * [Responsive Images](tutorials/core-concepts/09-working-with-templates/responsive-images.md)
+* [Apostrophe Search](tutorials/core-concepts/10-apostrophe-search/README.md)
+  * [Enabling Search](tutorials/core-concepts/10-apostrophe-search/search.md)
 
+## Advanced Topics
+* [Advanced Pieces Topics](tutorials/advanced-development/01-advanced-pieces-topics/README.md)
+  * [AJAX Features: Enhanced Browsing for Pieces](tutorials/advanced-development/01-advanced-pieces-topics/01-ajax-pieces.md)
+  * [Fine-grained Permissions for Pieces](tutorials/advanced-development/01-advanced-pieces-topics/02-permissions-for-pieces.md)
+  * [Adding Filters](tutorials/advanced-development/01-advanced-pieces-topics/04-adding-filters.md)
+  * [Adding Columns](tutorials/advanced-development/01-advanced-pieces-topics/05-adding-columns.md)
+  * [Using the Pieces Index](tutorials/advanced-development/01-advanced-pieces-topics/06-next-previous-links.md)
+  * [Adding Fields to All Pieces](tutorials/advanced-development/01-advanced-pieces-topics/07-adding-fields-to-all-pieces.md)
+  * [Adding new batch operations for pieces](tutorials/advanced-development/01-advanced-pieces-topics/08-adding-new-batch-operations.md)
+  * [Extending the Pieces Editor Modal](tutorials/advanced-development/01-advanced-pieces-topics/09-extending-the-pieces-editor-modal.md)
+* [Advanced Pages Topics](tutorials/advanced-development/02-advanced-pages-topics/README.md)
+  * [Custom Schemas for Pages](tutorials/advanced-development/02-advanced-pages-topics/customer-schemas-for-pages.md)
+* [How Apostrophe handles requests](tutorials/advanced-development/how-apostrophe-handles-requests.md)
+* [How Apostrophe modules are structured](tutorials/advanced-development/how-apostrophe-modules-are-structured.md)
+* [Promise Events](tutorials/advanced-development/03-promise-events/promise-events.md)
+* [Command line tasks](tutorials/advanced-development/04-apostrophe-cli/command-line-tasks.md)
+* [Internationalization and localization](tutorials/advanced-development/05-internationalization-localization/apostrophe-i18n-config.md)
+* [The Database](tutorials/advanced-development/06-database/README.md)
+  * [Apostrophe's model layer: working with the database](tutorials/advanced-development/06-database/01-model-layer.md)
+  * [Working with cursors](tutorials/advanced-development/06-database/02-cursors.md)
+  * [Accessing the database directly](tutorials/advanced-development/06-database/03-accessing-the-database-directly.md)
+  * [Using MongoDB replica sets with Apostrophe](tutorials/advanced-development/04-database/replica-set.md)
+* [Guide to schemas](tutorials/advanced-development/07-schemas/schema-guide/README.md)
```

Comment:
The schema guide is a big ol' reference in itself, so does it make sense for it to also be a parent page?

I would break this down as an overview with three child pages, one of which is the "reference guide to schema field types." You can get material for the overview from the top of the existing schema guide.

Reply:
Yeah. I think we still need to work on the positioning and framing for this a little bit.